ml_tools/eolearn/tests/test_sampling.py

```diff
@@ -184,28 +210,31 @@ def test_fraction_sampling_errors(fraction: Union[float, Dict[int, float]], repl
         )
 
 
+@pytest.fixture(name="fraction_task")
+def fraction_task_fixture(request) -> EOTask:
+    return FractionSamplingTask(
+        [(FeatureType.DATA, "NDVI", "NDVI_SAMPLED"), (FeatureType.MASK_TIMELESS, "LULC", "LULC_SAMPLED")],
+        (FeatureType.MASK_TIMELESS, "LULC"),
+        mask_of_samples=(FeatureType.MASK_TIMELESS, "sampling_mask"),
+        fraction=request.param[0],
+        exclude_values=request.param[1],
+    )
+
+
 @pytest.mark.parametrize("seed", range(3))
 @pytest.mark.parametrize(
-    "fraction, exclude",
+    "fraction_task",
     [
```

Comment:
```suggestion
    [  # fraction, exclude_values
```

just to know what the two parameters are